models/index.js

```diff
@@ -1,5 +1,6 @@
 const mongoose = require('mongoose');
-const User = require('./usersModel');
+
+const User = require('./usersSchema');
```

Comment:
I believe this should stay `./usersModel` to be consistent with other models file naming convention